include/zephyr/net/wifi_mgmt.h

```diff
@@ -229,6 +236,14 @@ enum net_event_wifi_cmd {
 #define NET_EVENT_WIFI_AP_STA_DISCONNECTED			\
 	(_NET_WIFI_EVENT | NET_EVENT_WIFI_CMD_AP_STA_DISCONNECTED)
 
+/** Wi-Fi version */
+struct wifi_version {
+	/** Driver version */
+	char *drv_version;
```

Comment:
Forgot to mention, these should be const pointers
```
const char *drv_version;
const char *fw_version;
```

Reply:
Done